client/js/table-view.js

```diff
@@ -59,12 +67,14 @@ class TableView {
         const position = {col: col, row: row};
```

Comment:
![screen shot 2017-05-03 at 4 10 58 pm](https://cloud.githubusercontent.com/assets/25340074/25685270/33592272-301b-11e7-9ab9-0ba29ccdae69.png)
this line is too indented?         const position = {col: col, row: row}; ? I'm not seeing why that is. do you want me to delete spaces?